ccp/index_reader.py

```diff
@@ -240,7 +242,7 @@ def apply_build_job(self,
         """
         Applies the build job template that creates pipeline to build
         image, and trigger first time build as well.
-        :param project: The name of project, where the template is to be applied
+        :param project: The name of project,where the template is to be applied
```

Comment:
This space is removed by error?

Reply:
No it was removed to fit the length of the string < 80 chars.

Reply:
Fixed.